oap-server/server-library/library-server/src/main/java/org/apache/skywalking/oap/server/library/server/http/HTTPServerConfig.java

```diff
@@ -39,4 +39,12 @@ public class HTTPServerConfig {
     private int acceptQueueSize = 0;
     @Builder.Default
     private int maxRequestHeaderSize = 8192;
+
+    @Builder.Default
+    private boolean enableTLS = false;
+
+    private int httpsPort;
```

Comment:
I am wondering are we going to share tls and no tls server together? If not, we could use port, rather than using a new port.
By this, I feel we open an insecure port for http.
It is better we don't expose that.

Reply:
For now I think use port is enough, because only `aws-receriver` need https, and it only support https.

So I modify now?

Reply:
Yes, please.